src/App.js

```diff
@@ -1,10 +1,108 @@
+import React from 'react';
+import './App.css';
+import { useState } from 'react';
+import TodoList from './components/TodoList';
+import TodoInsert from './components/TodoInsert';
+
+let nextId = 4;
+
+const App = () => {
+  const [todos, setTodos] = useState([
+    {
+      id: 1,
+      text: 'Todo 1',
+      isDone: false,
+    },
+    {
+      id: 2,
+      text: '할 일 2',
+      isDone: true,
+    },
+    {
+      id: 3,
+      text: 'long input test long input test long input test',
+      isDone: false,
+    },
+  ]);
+  const doneCnt = todos.filter((todo) => todo.isDone).length;
+  const todoCnt = todos.length - doneCnt;
+
+  const onInsertTodo = (text) => {
+    if (text === '') return;
+    const todo = {
+      id: nextId,
+      text,
+      isDone: false,
+    };
+    setTodos((todos) => todos.concat(todo));
```

Comment:
```suggestion
    setTodos((todos) => [ ...todos, todo ] );
```
ES6의 spread operator을 사용하면 concat함수를 사용하지 않고도 깔끔하게 값을 추가해줄 수 있습니다 :)
코드의 가독성이 좋아지는 건 덤입니다😁

Reply:
spread 연사자 사용이 자꾸 습관적으로 안나오네요,, 더 익숙해질 수 있도록 하겠습니다. 짚어 주셔서 감사합니다!